src/components/organisms/AdminVenueView/components/Spaces/Spaces.scss

```diff
@@ -0,0 +1,173 @@
+@import "scss/constants.scss";
+
+.Spaces {
+  display: flex;
+  flex: 1;
+
+  &__rooms {
+    width: 20%;
+  }
+
+  &__map {
+    width: 80%;
+    background-color: #19181a;
+    justify-content: center;
+    align-items: center;
+    padding: 40px;
+  }
+
+  &__background {
+    padding: 20px;
+  }
+
+  &__upload-background-wrapper {
+    background-color: #19181a;
+    border-radius: 16px;
+    padding: 40px;
+  }
+
+  &__upload-background-button {
+    background-color: rgba(255, 255, 255, 0.2);
+    border-radius: 26px;
+    padding: 10px;
+    text-align: center;
+  }
+
+  &__backgrounds {
+    display: flex;
+    position: relative;
+    overflow-x: scroll;
+  }
+
+  &__title {
+    font-size: 2rem;
+    font-weight: bolder;
+  }
+
+  &__venue-rooms {
+    padding: 10px;
+    background-color: $saturated-black;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border-bottom: 1px solid opaque-white(0.1);
+
+    &:hover {
+      background-color: lighten($saturated-black, 5%);
+    }
+  }
+
+  &__venue-room {
+    padding: $spacing--md;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    &:hover {
+      background-color: $saturated-black;
+    }
+  }
+
+  &__venue-room-logo {
+    height: 34px;
+    width: 34px;
+    margin-right: 10px;
+    background-size: contain;
+    background-position: center;
+    border-radius: 4px;
+    box-shadow: 0 0 0 3px $primary;
+  }
+
+  &__footer {
+    display: flex;
+    position: fixed;
+    bottom: 0px;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 20%;
+    padding: 12px;
+    background-color: #000;
+    border-top: 1px solid rgba(255, 255, 255, 0.2);
+  }
+
+  &__home-button {
+    cursor: pointer;
+    display: flex;
+    width: 50px;
+    height: 50px;
```

Comment:
home-button-size

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553